react/Checkbox/Checkbox.test.js

```diff
@@ -1,126 +1,61 @@
-import chai, { expect } from 'chai';
-import sinon from 'sinon';
-import sinonChai from 'sinon-chai';
 import React from 'react';
-import {
-  Simulate,
-  renderIntoDocument,
-  scryRenderedDOMComponentsWithClass,
-  findRenderedDOMComponentWithClass
-} from 'react-dom/test-utils';
-import { createRenderer } from 'react-test-renderer/shallow';
-import SyntheticEvent from 'react-dom/lib/SyntheticEvent';
-import { findAllWithClass } from 'react-shallow-testutils';
-import Checkbox from './Checkbox';
-
-chai.use(sinonChai);
-
-const eventMatcher = sinon.match.instanceOf(SyntheticEvent);
+import { shallow } from 'enzyme';
```

Comment:
Yep I get that and agree that that's the best way for snapshots to be formatted.  In profile we explicitly mock out any subcomponents like this `jest.mock('./ResumeItem/ResumeItem', () => 'ResumeItem');` rather than doing a shallow render.  I prefer to do the mocking manually so that it's clear which sub-components are being mocked and it gives you the flexibility mock some sub components and not others